frontend/components/campaigns/CampaignDescription.tsx

```diff
@@ -0,0 +1,30 @@
+import { Typography } from '@mui/material';
+import React from 'react';
+import { CampaignPublicData } from '../../types/campaigns';
+import CloseQuotes from '../icons/CloseQuotes';
+import OpenQuotes from '../icons/OpenQuotes';
+
+type Props = {
+  campaign: CampaignPublicData;
+};
+
+export default function CampaignDescription({ campaign }: Props) {
+  return (
+    <>
+      <Typography variant="h2">About the campaign</Typography>
+
+      <Typography variant="body1">
+        This campaign was started by <strong>{campaign.primaryDonor.name}</strong> who has generously committed
+        <strong> ${campaign.promisedAmount}</strong> to the beneficiaries listed above.
+      </Typography>
+
+      <OpenQuotes />
```

Comment:
Can we try using this instead? It's easier to indicate the colors

`import FormatQuoteIcon from '@mui/icons-material/FormatQuote';`
https://mui.com/material-ui/material-icons/?query=quote&selected=FormatQuote

You likely can flip the quote with sxProps